src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let cloneShadow = {};
```

Comment:
It's a little bit strange for me and it looks unnecessary, to create a variable for an empty object for Object.assign in the loop below. In my opinion, it seems better just to add an empty object as an argument for the method, not as a variable. We need to ask a mentor.